src/simulation/include/inline_riemann.fpp

```diff
@@ -26,10 +28,39 @@
     gamma_avg = (sqrt(rho_L)*gamma_L + sqrt(rho_R)*gamma_R)/ &
                 (sqrt(rho_L) + sqrt(rho_R))
 
-    rho_avg = sqrt(rho_L*rho_R)
     vel_avg_rms = (sqrt(rho_L)*vel_L(1) + sqrt(rho_R)*vel_R(1))**2d0/ &
                   (sqrt(rho_L) + sqrt(rho_R))**2d0
 
+    #:if chemistry
```

Comment:
is there a reason we use fypp macros here instead if just `if` statements? it does make doing static code analysis harder, since the generated code diverges depending on options.

Reply:
We do this because of a _"limitation with Pyrometheus"_. Indeed, we cannot generate a thermochem file for chemistry if there is no chemistry. So a bunch of symbols would be missing if we didn't use `#:if chemistry`. I see no reason why we couldn't generate a thermochem file for a random mechanism if chemistry is off. This could be a GH issue @sbryngelson.

Reply:
I see. It does seem worth creating a `null` reaction mechanism that makes the symbols exist but otherwise does no compute and minimal (or no?) allocations. Can be a GH issue.

Reply:
We can actually use any reaction mechanism since it will never be used by MFC

Reply:
closed in lieu of this issue https://github.com/MFlowCode/MFC/issues/678